data/map planets.txt

```diff
@@ -159,10 +161,11 @@ planet Venus
 		fleet "Large Republic" 14
 
 planet "New Sichuan"
-	attributes "near earth" tourism urban
+	attributes "near earth" tourism urban farming
 	landscape land/ncity10
-	description "New Sichuan was one of the first Earthlike worlds discovered by Humans, a steep mountain world, livestock was discarded in favor of farming rice in gardens sculpted from the rock of the mountains. as the centuries passed the cities grew and farms became more automated. The day interstellar commerce broke down, food became more scarc and the population suffered famines resulting in overproduction ravaging the planet, an exodus ocurred and the cities became desolated. "
-	description "When trade resumed again population normalized, however measures to make sure the tragedy never repeated led to the construction of large towers that house aquoponic farms for meat and other vegetables, today those towers became simbolic of the Human ability to adapt to the challenges and recieve many visitors."
+	description "New Sichuan was one of the first Earth-like worlds settled by humans. A steep mountain world, the initial colonists initially introduced livestock, but came to favor farming rice in the terraces sculpted from the rock of the mountains. As the centuries passed the cities grew to become some of the largest in human space and the farms became more automated, creating an unique system of farming complexes and distribution centers that efficiently provided for the burgeoning planet, as well as other planets in nearby systems."
+	description "	During the Alpha Wars, New Sichuan was briefly occupied by the Alphas. In their retreat from Human forces, they systematically destroyed the farming complexes and the distribution centers, leading to widespread famine across New Sichuan. Many cities were deserted for the safety of better protected planets like Earth or Haze."
+	description "	When the Alpha Wars subsided and the population of the planet was largely restored during the Caravan Era, measures to taken to ensure the tragedy was never lest the Alphas rose again. Each city holds large towers housing aquoponic farms for meat and other vegetables, in order to supplement the fertile rice fields that New Sichuan still holds and exploits; today these towers are symbols of the human ability to adapt in the face of mounting challenges, and recieve many off-world visitors."
 	spaceport "New Sichuan Starport is unique among ports as its carved on the face of a mountain. The massive complex is made out of holes in the stone wall, extending deep underground for thousands of ships."
 	spaceport "The hub is on the side facing one of the few valleys in this mountain range where the capital resides. Balconies with thousands of people admire the view, take photos, etc."
```

Comment:
It's not actually that unique, other spaceports like Delve and Ada are carved out of the side of cliff faces.